great_expectations/data_context/templates.py

```diff
@@ -76,146 +90,61 @@
       class_name: FixedLengthTupleFilesystemStoreBackend
       base_directory: expectations/
 
-  local_validation_result_store:
+  validations_store:
     class_name: ValidationResultStore
     store_backend:
       class_name: FixedLengthTupleFilesystemStoreBackend
       base_directory: uncommitted/validations/
       filepath_template: '{4}/{0}/{1}/{2}/{3}.json'
 
-  # Uncomment the lines below to enable s3 as a result store. When enabled,
-  # validation results will be saved in the store according to `run id`.
-  # For S3, ensure that appropriate credentials or assume_role permissions are
-  # set where validation happens.
-  
-  # s3_validation_result_store:
-  #   class_name: ValidationResultStore
-  #   store_backend:
-  #     class_name: FixedLengthTupleS3StoreBackend
-  #     bucket: <my_bucket>
-  #     prefix: <my_prefix>
-  #     file_extension: json
-  #     filepath_template: '{4}/{0}/{1}/{2}/{3}.{file_extension}'
-
-
```

Comment:
Bye bye non working code

Reply:
👍 (though I think that code does work if uncommented, right????)

Reply:
I do not know, I have not tested it.

Reply:
rgr. Confirmed that it does not. But I'll address that separately.